recipes/soxr/all/conanfile.py

```diff
@@ -1,7 +1,9 @@
-import functools
 import os
-from conans import ConanFile, CMake, tools
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, load, save, rmdir
 from conan.tools.microsoft import msvc_runtime_flag, is_msvc
+from conan.tools.scm import Version
 
 
 required_conan_version = ">=1.45.0"
```

Comment:
Lots of good updates here

```suggestion
required_conan_version = ">=1.53.0"
```